engine/src/engine.lcb

```diff
@@ -176,6 +178,7 @@ end syntax
 /**
 Summary:	The property of a script object.
 Property: 	The name of the property to manipulate
+Set:		The name of the custom property set to access
```

Comment:
@livecodeian doesn't this need to be moved down to a doc block for `PropertyOfSetOfScriptObject`?

Reply:
@livecodeian did the new syntax need to be documented separately or is this fine as is?